src/modules/extra/m_ssl_openssl.cpp

```diff
@@ -204,6 +204,47 @@ namespace OpenSSL
 			return SSL_CTX_load_verify_locations(ctx, filename.c_str(), 0);
 		}
 
+		void SetCRL(const std::string& crlfile, const std::string& crlpath, const std::string& crlmode)
+		{
+			unsigned long crlflags;
+			X509_STORE* store;
```

Comment:
This should be declared where it is actually used (just before 223?) rather than at the top.